apps/dashboard/pages/api/logs/index.ts

```diff
@@ -47,14 +63,24 @@ export const getLogs = async (req: AppNextApiRequest, res: NextApiResponse) => {
       messages: {
         take: 1,
         orderBy: {
-          createdAt: 'asc',
+          createdAt: 'asc' as Prisma.SortOrder,
         },
       },
     },
     orderBy: {
-      updatedAt: 'desc',
+      updatedAt: 'desc' as Prisma.SortOrder,
     },
-  });
+  };
+
+  if (filter) {
+    const conversations = await prisma.conversation.findMany({
```

Comment:
Please use conditional filters like here 
https://github.com/gmpetrov/databerry/blob/main/apps/dashboard/pages/api/conversations/index.ts
I find it less error-prone + no code duplication